package.json

```
@@ -77,7 +77,7 @@
     "vue-router": "^3.6.5",
     "vue-tiny-slider": "^0.1.39",
     "vuex": "^3.6.2",
-    "youtubei.js": "^5.1.0"
+    "youtubei.js": "5.2.0"
```

Please keep the `^` prefix to avoid code conflict (and allow dependency auto update by bots) :)
```suggestion
    "youtubei.js": "^5.2.0"
```